clientlibs/java/pom.xml

```
@@ -20,10 +20,10 @@
 		</plugins>
 	</build>
 	<properties>
-		<jersey.version>2.16</jersey.version>
+		<jersey.version>2.30.1</jersey.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.source>1.8</maven.compiler.source>
-		<maven.compiler.target>1.8</maven.compiler.target>
+		<maven.compiler.source>17</maven.compiler.source>
+		<maven.compiler.target>17</maven.compiler.target>
```

They unified `source` and `target` into a new `release` parameter:

```suggestion
		<maven.compiler.release>17</maven.compiler.release>
```